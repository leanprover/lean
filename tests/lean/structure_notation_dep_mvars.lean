instance : lawful_monad list := { .. }
