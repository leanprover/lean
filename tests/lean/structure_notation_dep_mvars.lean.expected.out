structure_notation_dep_mvars.lean:1:32: error: type mismatch at field 'pure_bind'
  ?m_1
has type
  ∀ {α β : Type ?} (x : α) (f : α → list β), ?m_1 (?m_2 x) f = f x
but is expected to have type
  ∀ {α β : Type ?} (x : α) (f : α → list β), pure x >>= f = f x
